Yoast/wpseo-news
# Move each class to its own file.

## Summary

This PR can be summarized in the following changelog entry:
* Move each sitemap related class to its own file instead of having multiple classes in one single file.

## Test instructions

This PR can be tested by following these steps:

* Nothing functional change. Please verify if the sitemap still works.

Fixes #
